packages/integration-tests/src/EntityManager.test.ts

```diff
@@ -757,6 +756,18 @@ describe("EntityManager", () => {
     expect(rows.length).toEqual(0);
```

Comment:
Should this test assert that the beforeFlush hook was run? I.e. right now the cascade delete could be happening via a db fk cascade and the test would still pass / not realize that it's not our beforeFlush that is doing the cascade.